hologram_dashboard/fetcher/BUILD

```diff
@@ -0,0 +1,19 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "fetcher_main",
+    srcs = ["fetcher_main.cc"],
+    deps = [
+        "hologram_data_source_availabiliity_fetcher"
+    ],
+)
+
+cc_library(
+    name = "hologram_data_source_availabiliity_fetcher",
+    srcs = ["hologram_data_source_availabiliity_fetcher.cc"],
+    hdrs = ["hologram_data_source_availability_fetcher.h", 
+            "hologram_data_fetcher.h"],
+    deps = ["//fetcher/proto:hologram_availability_cc_proto", 
+            "//fetcher/proto:hologram_config_cc_proto", 
+            "@com_github_gflags_gflags//:gflags"],
```

Comment:
You need to define cc_library for hologram_data_fetcher.h by itself, and depends on it under hologram_data_source_availabiliity_fetcher.